src/main/java/org/radarcns/management/service/dto/SourceDTO.java

```diff
@@ -13,8 +13,7 @@
 
     private Long id;
 
-    @NotNull
-    private String sourceId;
+    private UUID sourceId;
```

Comment:
if we gonna have `UUID` in SourceDTO which we need to change the typescript to receive `UUID` type. The `Source id` text box should be removed when from `general.subject-dialog.component.html` as we agreed that we will not enter a `UUID` manually.

Reply:
Do you mean from `general.source-dialog.component.html` instead? It is removed there already, or rather, it will be shown only, and read-only, when it is already part of the model, i.e. when editing an existing source.
The serialization and de-serialization of the DTO takes care of converting from and to the appropriate string representation of a `UUID` type.

Reply:
indeed. it was not the `subject`. But what i meant was the `app/shared/sources/source-dialog-component.html`.  But it is not very user-friendly to enter an UUID manually. Isn't it